index.js

```diff
@@ -111,20 +142,34 @@ class RPFormatter extends Formatter {
         }))
     }
 
-    getMessage(step) {
-        if (!step.pickle) return 'Hook';
+    getStepText(step, steps) {
+        if (!step.pickle) return this.hookKeyword(step, steps);
         const messageParts = [step.pickle.text];
         if (step.pickle.argument) {
             if (step.pickle.argument.dataTable) messageParts.push(
                 this.formatTable(step.pickle.argument.dataTable)
             )
             if (step.pickle.argument.docString) messageParts.push(this.formatDocString(step.pickle.argument.docString))
         }
-        if (step.result.status === Status.FAILED) messageParts.push(step.result.message)
 
         return messageParts.join('\n')
     }
 
+    hookKeyword(step, steps) {
+        const stepsBefore = steps.slice(0, steps.findIndex((element) => element === step));
+        return stepsBefore.every(element => element.pickle === undefined) ? 'Before' : 'After'
```

Comment:
I've noticed missing semicolons at the end of lines across all changed files.

Reply:
I do not add semicolons for return statements